test_mcp_server.py

```
@@ -0,0 +1,172 @@
+        return True
+        
+    except Exception as e:
+        print(f"✗ Test failed with error: {e}")
+        return False
+        
+    finally:
+        # Clean up
+        try:
+            process.terminate()
+            process.wait(timeout=5)
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)